pkg/meta/meta.go

```diff
@@ -1012,6 +1014,68 @@ func (m *Meta) GetMetasByDBID(dbID int64) ([]structure.HashPair, error) {
 	return res, nil
 }
 
+// CheckSpecialAttributes checks if the special attributes are in the table info.
+// Make it same as hasSpecialAttributes.
+// Exported for testing.
+func CheckSpecialAttributes(str string) bool {
+	if strings.Contains(str, "\"tiflash_replica\":null") && strings.Contains(str, "\"policy_ref_info\":null") &&
```

Comment:
Will the json object skip displaying the null field?
For example, the table info has not partition, so instead of an explicit "partition:null", there is nothing.

Reply:
No, it's tested and I don't think it would change.